### DISCUSSION TITLE
Incorporating / dealing with Behave test suites in a pytest framework

### DISCUSSION THREAD

**Discussion by robbybbb** [pytest-dev/pytest#10736] [Q&A]
We are a pytest shop with a lot of tests.  We are picking up software and incorporating it into our product that comes with a large suite of Behave test cases.  One thing that is off the table is moving from pytest to Behave.  We are firmly remaining with pytest.  We'd like to converge on using the pytest framework for running everything.  Any advice on what to do would be appreciated.  If anybody has any experience or tools to help with Behave tests in a pytest framework it would be much appreciated.

**Comment by The-Compiler** [comment]
[pytest-dev/pytest-bdd: BDD library for the py.test runner](https://github.com/pytest-dev/pytest-bdd) should be reasonably close syntax-wise hopefully.

**Reply by robbybbb** [reply]
That's what we are hoping for too.  Let's see if there's anybody out there with experience and/or tools.  Thanks!
